source/services/facebook.js

```diff
@@ -1,13 +1,20 @@
 /**
  * Facebook service provider
+ * Share doc: https://developers.facebook.com/docs/workplace/sharing/share-dialog/
+ * Counter doc: https://developers.facebook.com/docs/graph-api/reference/v8.0/engagement
+ *
+ * For hackers: the access token below is public, heavily restricted and doesn't allow to access anything of value
  */
 
 export default {
-    counterUrl: 'https://graph.facebook.com/?id={url}&fields=og_object%7Bengagement%7Bcount%7D%7D&callback=?',
-    convertNumber: (data) => data.og_object.engagement.count,
-    popupUrl: 'https://www.facebook.com/sharer/sharer.php?u={url}',
-    popupWidth: 600,
-    popupHeight: 500,
-    knownParams: ['url'],
-    svgIconPath: '15.117 0H.883C.395 0 0 .395 0 .883v14.234c0 .488.395.883.883.883h7.663V9.804H6.46V7.39h2.086V5.607c0-2.066 1.262-3.19 3.106-3.19.883 0 1.642.064 1.863.094v2.16h-1.28c-1 0-1.195.48-1.195 1.18v1.54h2.39l-.31 2.42h-2.08V16h4.077c.488 0 .883-.395.883-.883V.883C16 .395 15.605 0 15.117 0',
+    counterUrl: 'https://graph.facebook.com/?id={url}&access_token=1729830587180291|102e6d79cda2fa63b65c99c039eed12a&fields=og_object%7Bengagement%7Bcount%7D%7D',
```

Comment:
This is an application token, but it's not secret and is limited by 200 per day for each IP. 
Facebook does not support the necessary API call if the app does not identify itself.